.cursor/rules/$c.sbm.md

```
@@ -0,0 +1,80 @@
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)